reservation/tests.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+from reservation.models import TypeOfJob, Appointment, Schedule
+from datetime import datetime
+
+PROFESSIONAL_ID = 4
+TYPEOFJOB_NAME = "Gel nail polish"
+PRICE = 90
+
+CLIENT_ID = 3
+START_APPOINTMENT = datetime(2023, 4, 17, 12, 0, 0)
+END_APPOINTMENT = datetime(2023, 4, 17, 13, 0, 0)
+SUMMARY = ""
+
+START_DAY = datetime(2023, 4, 17, 10, 0, 0)
+END_DAY = datetime(2023, 4, 17, 18, 0, 0)
+MEETING_TIME = 60
+
+
+@pytest.fixture
+def typeOfJob():
+    return TypeOfJob(professional_id=PROFESSIONAL_ID, typeOfJob_name=TYPEOFJOB_NAME, price=PRICE)
+
+
+@pytest.fixture
+def persisted_typeOfJob_pool(typeOfJob):
```

Comment:
the typeOfJob -> consider ranaming to type_of_job

Reply:
Done